### PR TITLE
Fix logging across scilpy

### PR DESCRIPTION
**Pull request by frheault**
Thanks to Laurent I realized that `scil_search_keywords.py` stopped working.
It was due to the fact that setting logging to `INFO` or `DEBUG` the way we did (everywhere in Scilpy) does not work anymore.

So it also meant that around 70% of our logging was not working properly (WARNING is fine, because it was the default for the `root::logging`).
So I fixed it everywhere.

I also added a `pytest.ini` to print warnings from other libraries **only once** and keep all warnings from Scilpy (**even if many**) reducing printed line from 1M to 

PS: A few pep8 changes got in there while saving files, a few scripts didn't have permissions like everything else (777) so I changed it for uniformity.

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by frheault** [comment]
So the total output of tests on Travis is now less than 1000 (including the checkout, build, tests and warning)
`= 368 passed, 76 warnings in 2029.82s (0:33:49) =`

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_run_commit.py [hunk 1]
```diff
@@ -80,8 +79,7 @@
 import numpy as np
 import nibabel as nib
 
-from scilpy.io.streamlines import (lazy_streamlines_count,
```
Comment (AntoineTheb) [inline scripts/scil_run_commit.py:83]: Why is `lazy_streamlines_count` no longer imported ? Was it unused before ?

**Review by AntoineTheb — APPROVED** [review]
One comment but otherwise LGTM !

**Review by EmmaRenauld — APPROVED** [review]
I tested the pytest locally and, man!! Nice! So much easier! Takes a lot less time, too.

File: scripts/scil_run_commit.py (hunk 1 above)
Comment (frheault) [reply scripts/scil_run_commit.py:83]: I am setting up my new computer and my autopep8 settings had too much power I think.

**Pull request merged**